docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -127,17 +150,72 @@ PAF-of-1 Cause/Risk Pairs
-++++++++++++++++++++++
+Stratifying Exposure Variables by Baseline Intervention Coverage
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+From GBD we obtain overall population estimates for our outcomes of interest
+(prevalence of vitamin A deficiency, birth prevalence of neural tube defects,
+distribution of haemoglobin levels, and distribution of birth weights). Because
+there exists baseline coverage of our interventions --- i.e. there is already
+some food fortification occurring in the locations we are simulating --- these
+overall population estimates include both people who are eating fortified food
+and those who are not.
+
+When we introduce additional food fortification into the population, only those
+who are *not* already eating fortified food should receive the benefit.
+Therefore, in order to properly measure the effects of our interventions, we
+must first stratify our population into those who are already covered vs. not
+covered by the forticant of interest. We then need to calculate the risk
+exposure (or other relevant outcome) in each coverage stratum to get a complete
+picture of the baseline population. For example, we will need to know the
+prevalence of vitamin A deficiency or the distribution of haemoglobin in the
+subpopulation already eating fortified food as well as in the subpopulation not
+eating fortified food.
+
+These coverage-stratified exposures are calculated so that:
+
+1.  Their population-weighted average matches the overall population estimate
+    from GBD, and
+2.  The differential exposure between the subpopulations matches the
+    corresponding effect size for our intervention.
+
+For example, the ratio of the prevalence of vitamin A deficiency between the
+fortified and unfortified groups should match the risk ratio of vitamin A
+fortification on vitamin A deficiency from the literature, and the difference in
+mean haemoglobin level between the fortified and unfortified groups should match
+the mean shift in haemoglobin from iron fortification found in the literature.
+When we stratify the population in this way, we are assuming that the effect
+sizes of fortification found in the literature are generalizable to the
+populations we are simulating (note that if this assumption is *not* true, we
+should not be simulating the interventions in theses populations at all).
+
+.. todo::
 
-Effect size stratification for baseline population
---------------------------------------------------
+  Come up with some examples to show what would happen if we *didn't* first
+  stratify by existing coverage. I.e., why is this baseline stratification
+  important? It is not totally obvious to me what the effect would be if we
+  didn't do it (e.g. would we end up overestimating or underestimating the true
+  effect?) -- I think the net effect depends on the level of existing coverage,
+  the shape of the underlying exposure distribution, and the magnitude of the
+  effect size.
 
```

Comment:
I think that the most obvious example of this is for birth weight. The additional 10 grams of birth weight from an intervention will be much more valuable to you if you are born in the 0-500 gram category than the 2500 to 2750 category. 

So, if we DO calibrate to baseline coverage, the uncovered group will have a lower baseline mean birth weight (say 2500) than the covered group (say 2750). The difference in RR between the 2500 g group and the next birth weight group >> the difference in RR between the 2750 group and the next birth weight group. Essentially, we are helping those who really need it!

If we do NOT calibrate for baseline coverage, both groups have the same starting mean birth weight. In this case, the uncovered group will be starting slightly "better off" and therefore have less to gain from the intervention (we might underestimate the effect of the intervention).

A similar situation is true for dichotomous exposures like VAD. If overall population prevalence of VAD is 20% (baseline covered VAD prevalence = 10%, baseline uncovered = 30%) and the RR for vitamin A fortification is 0.5. Also let's say that baseline coverage is 50%. 

So if we DO account for baseline coverage differences, we will reduce VAD by 50% in the baseline uncovered group... 30% to 15% so that 15% of the population remits.

If we DO NOT accoun for bsaeline coverage differences, we will still reduce VAD by 50% in the baseline uncovered group, but there will be *less* VAD there to reduce so that VAD prevalence changes from 20% to 10% so that 10% of the population remits.

Reply:
Cool, thanks Ali.  I'll paste this into the todo so we can edit it later.